### PR TITLE
Extend actions/events testing with cloud providers

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/control/test_actions.py -v --long-running --use-provider complete}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6dd6010d6f108d2ad538ca5323a30c9f862e4701:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 27727d31ae503219cae138d9a9eaf83b4b3db3d8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6f410f4ccbdb8466b837aee0fd6ac2ea987a144d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 75a29a949dddf0a6b48cebb2482c2515262671e8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py [hunk 1]
```diff
@@ -196,8 +204,16 @@ def automate_role_set(request):
...
 
 @pytest.fixture(scope="module")
-def vm_crud(vm_name, provider_crud):
-    return Vm(vm_name, provider_crud)
+def vm_crud(vm_name, provider_crud, provider_key, provider_type):
+    if is_cloud_provider(provider_key):
+        if provider_type == "openstack":
+            return OpenStackInstance(vm_name, provider_crud)
+        elif provider_type == "ec2":
+            return EC2Instance(vm_name, provider_crud)
```
Comment (seandst) [inline cfme/tests/control/test_actions.py]: Is there any hope for unifying the Instance types so we don't need this check?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/control/test_actions.py (hunk 1 above)
Comment (mfalesni) [inline cfme/tests/control/test_actions.py]: I have a branch somewhere, where I tried to do that, having the same base for both VM and instances, unifying and DRYing the code.

**Comment by dajoRH** [comment]
Lint report for commit ee1030398c651461ca96f20b9ad4509e24105edc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a822df8eab95330ba4af3032765b52484a40caa6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py (hunk 1 above)
Comment (seandst) [inline cfme/tests/control/test_actions.py]: If that doesn't work out, I think I'd be alright if we just got all the cloud providers merged for now.

**Pull request merged**